docs/api/spaces-management/post.asciidoc

```diff
@@ -30,7 +30,7 @@ The following parameters can be specified in the body of a POST request to creat
 `initials`:: (string) Optionally specify the initials shown in the Space Avatar for this space. By default, the initials will be automatically generated from the space name.
 If specified, initials should be either 1 or 2 characters.
 
-`color`:: (string) Optioanlly specify the hex color code used in the Space Avatar for this space. By default, the color will be automatically generated from the space name.
+`color`:: (string) Optionally specify the hex color code used in the Space Avatar for this space. By default, the color will be automatically generated from the space name. **NOTE**: The hex color code must be all lowercase.
```

Comment:
Instead of starting with `Optionally`, you can just use `(Optional, string)`

So the rest of the sentence would be:

`Specifies the hex color code...`

Asciidoc automatically renders NOTES in bold. I would change this to `NOTE:` or you might run into some weird formatting.